src/parser/jobs/sam/modules/OGCDDowntime.js

```diff
@@ -0,0 +1,14 @@
+import ACTIONS from 'data/ACTIONS'
+import CooldownDowntime from 'parser/core/modules/CooldownDowntime'
+
+export default class OGCDDowntime extends CooldownDowntime {
+	//Time that samurais have deemed ok for a OGCD to be down
+	// eslint-disable-next-line no-magic-numbers
+	allowedDowntime = 5000
+	trackedCds = [
```

Comment:
You need to add a firstUseOffset as well - the CooldownDowntime module doesn't consider allowedDowntime as delay from pull to first use, so pulls that are just longer than an even multiple of a cooldown will have funky expected uses if you don't specify a firstUseOffset.